.dockerignore

```diff
@@ -0,0 +1,6 @@
+# Ignore these directories when building images. Note some may be populated anyway on composer install.
```

Comment:
the file name should be `.dockerignore` - note the dot

Reply:
In the PR is it with a dot. Not sure why it's named differently here. See https://github.com/simesy/govcms8-scaffold-paas/blob/master-si/.dockerignore